ember/app/controllers/map.js

```diff
@@ -138,12 +140,23 @@ export default class extends Controller {
     );
   }
 
+  @action
+  setBaseMap(baseMap) {
+    this.get('map').set('baseMap', baseMap);
+    this.set('baseMap', baseMap);
```

Comment:
Since both the controller's `baseMap` and `map.baseMap` hold the same value, you can just alias `map.baseMap` from the controller: 

```js
@alias('map.baseMap') baseMap
```

This means that you can delete line 146 because `baseMap` will just be whatever you set `map.baseMap` to.

http://ember-decorators.github.io/ember-decorators/latest/docs/api/modules/@ember-decorators/object/computed#alias